tasks/copy_to_glacier/test/unit_tests/test_copy_to_glacier.py

```diff
@@ -331,6 +331,85 @@ def test_task_overridden_multipart_chunksize(self):
         self.assertEqual(expected_granules, result["granules"])
         self.assertIsNone(config_check.bad_config)
 
+    @patch.dict(
+        os.environ,
+        {
+            "ORCA_DEFAULT_BUCKET": uuid.uuid4().__str__(),
+            "DEFAULT_MULTIPART_CHUNKSIZE_MB": "4",
+        },
+        clear=True,
+    )
+    def test_task_overridden_default_bucket(self):
+        """
```

Comment:
This test works, just thinking out of the box a little bit, I wonder if it would be easier to throw the logic for setting the glacier bucket into a small function like `get_orca_archive_bucket(config)` that would return the proper bucket name. This would then simplify the test. That function then could be done throughout the other tasks with the same test essentially.

Reply:
Holding off due to potential merge conflicts.